test/old/test.js

```diff
@@ -0,0 +1,42 @@
+const Tonic = require('@optoolco/tonic')
```

Comment:
This seems like it could be killed off.

Reply:
This file creates the old test.html page if you want to keep that workflow ( npm run dev and go to localist:8080/test.html )